src/VisualStudio/LiveShare/Impl/CustomProtocol/Classifications.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.VisualStudio.LanguageServices.LiveShare.CustomProtocol
+{
+    // TODO - Move this to LSP extensions or elsewhere.
+    // Temporary internal copy so we can implement these custom extensions
+    // until these types are available elsewhere.
+    internal class ClassificationParams
```

Comment:
I guess we need work item for classification

Reply:
https://github.com/dotnet/roslyn/projects/45#card-20619634